### PR TITLE
Increase the size of the first item on the Home tab.

### LINKED ISSUE #37: Increase the size of the first item on the Home tab.
## Kind (Required)

- Improvement

## Overview (Required)

- The title says everything

image

 ## Links	

https://www.figma.com/file/IFlrbfmBSdYvUz7VmSzfLV/DroidKaigi_2021_official_app
There are some parts where the design size is not accurate. Use 4dp * x size and similar Material components.
Please log in to see the size on the design.

### PR DESCRIPTION
**Pull request by hiesiea**
## Issue
- close #37 

## Overview (Required)
- Implements FirstFeedItem.
- Fix to show FirstFeedItem when the first item on the Home tab.

## Links
- https://www.figma.com/file/IFlrbfmBSdYvUz7VmSzfLV/DroidKaigi_2021_official_app

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

File: uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedScreen.kt [hunk 1]
```diff
@@ -43,7 +43,11 @@ import kotlin.reflect.KClass
 
 sealed class FeedTabs(val name: String, val routePath: String) {
     object Home : FeedTabs("Home", "home")
-    sealed class FilteredFeed(val feedItemClass: KClass<out FeedItem>, name: String, routePath: String) :
+    sealed class FilteredFeed(
+        val feedItemClass: KClass<out FeedItem>,
```
Comment (takahirom) [inline uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedScreen.kt:47]: 👍

File: uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FirstFeedItem.kt [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+        MaterialTheme.colors.secondary
+    }
+    else -> {
+        Color.Gray
+    }
+}
+
+@Preview(showBackground = true)
+@Composable
+fun PreviewFirstFeedItem() {
```
Comment (takahirom) [inline uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FirstFeedItem.kt:152]: 🆒

**Review by takahirom — APPROVED** [review]
LGTM 👍

**Comment by takahirom** [comment]
Sorry. Can you resolve conflict?

**Comment by hiesiea** [comment]
@takahirom 
I've resolved the conflicts.

**Review by takahirom — APPROVED** [review]
Thank you for your cooperation!

**Pull request merged**